packages/charts-line/index.html

```diff
@@ -22,6 +22,29 @@
               },
               id: 'myChart',
             });
+
```

Comment:
The code below only demonstrates the lines updating every 5 seconds. I thought this was important to show how you call a function on the chart: `ibmChart.uniqueid`